readme.md

```diff
@@ -1,38 +1,54 @@
 
-*Files to look at*:
+This project contains the custom dashboard items that you will get after you complete the [Custom Item](https://docs.devexpress.com/Dashboard/403031/winforms-dashboard/winforms-designer/ui-elements-and-customization/create-a-custom-item) tutorials.
 
-* [Form1.cs](./CS/TutorialsCustomItems/Form1.cs) (VB: [Form1.vb](./VB/TutorialsCustomItems/Form1.vb))
-* [Modules](./CS/TutorialsCustomItems/CustomItems/) (VB: [Modules](./VB/TutorialsCustomItems/CustomItems/))
+## Static Custom Item
+
+**View Tutorial**: [Create a Static Custom Item](https://docs.devexpress.com/Dashboard/403087/winforms-dashboard/winforms-designer/ui-elements-and-customization/create-a-custom-item/create-a-static-item)
+
+**View File**: [С#](./CS/TutorialsCustomItems/CustomItems/)/ [VB](./VB/TutorialsCustomItems/CustomItems/)
+
+The file contains contains a code for a **HelloWorld** custom item that displays the 'Hello World!' text. 
+
+![](images/win-dashboard-static-custom-item.png)
 
+## Data-Aware Custom Item
 
-This project contains the following custom dashboard items that allow you to add additional functionality to the WinForms Dashboard:
+**View Tutorial**: [Create a Data-Aware Item](https://docs.devexpress.com/Dashboard/403088/winforms-dashboard/winforms-designer/ui-elements-and-customization/create-a-custom-item/create-a-data-aware-item)
 
-* **Hello World** ([С#](./CS/TutorialsCustomItems/CustomItems/)/ [VB](./VB/TutorialsCustomItems/CustomItems/))
+**View File**: [С#](./CS/TutorialsCustomItems/CustomItems/)/ [VB](./VB/TutorialsCustomItems/CustomItems/)
 
-    A custom item that displays the 'Hello World!' text. 
+The file contains a code for a custom item based on the [Grid](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.GridControl) control.
 
-* **Simple Table** ([С#](./CS/TutorialsCustomItems/CustomItems/)/ [VB](./VB/TutorialsCustomItems/CustomItems/))
+![](images/win-dashboard-data-aware-custom-item-with-binding-panel.png)
 
-    A custom item based on the [Grid](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.GridControl) control.
+## Data-Aware Item based on External Visualization Widget
 
-* **Funnel Chart** ([С#](./CS/TutorialsCustomItems/CustomItems/)/ [VB](./VB/TutorialsCustomItems/CustomItems/))
+**View Tutorial**: [Create an Interactive Data-Aware Item](https://docs.devexpress.com/Dashboard/403032/winforms-dashboard/winforms-designer/ui-elements-and-customization/create-a-custom-item/create-an-interactive-data-aware-item)
 
-    A Funnel chart displays a wide area at the top, indicating the total points' value, while other areas are proportionally smaller.
+**View File**: [С#](./CS/TutorialsCustomItems/CustomItems/)/ [VB](./VB/TutorialsCustomItems/CustomItems/)
+
+The file contains a code for a more "real-world" custom item that uses an external visualization control. A Funnel chart displays a wide area at the top, indicating the total points' value, while other areas are proportionally smaller.
```

Comment:
```suggestion
The file contains a code for a more "real-world" custom item that uses an external visualization control. A Funnel chart displays a wide area at the top that indicates the total points' value, while other areas are proportionally smaller.
```

Reply:
Just to clarify, by `the total points' value` do you mean the value of the total points?